lib/parliament/request/base_request.rb

```diff
@@ -101,7 +106,7 @@ class << self
       end
 
       def build_response(net_response)
-        @builder.new(net_response).build
+        @builder.new(response: net_response, decorators: @decorators).build
```

Comment:
If we are calling like this our base builder should either do something with the decorators or actively ignore them?